tests/test_georaster.py

```diff
@@ -95,8 +95,9 @@ def test_info(self) -> None:
             "transform",
             "width",
         ]
-        for attr in default_attrs:
-            assert r.__getattribute__(attr) == r.ds.__getattribute__(attr)
+        with rio.open(datasets.get_path("landsat_B4")) as dataset:
```

Comment:
This is minor, but to be sure it's not broken in the future, maybe the path should be saved in a variable, and the same variable used at this line and L 80?

Reply:
Changed in 74b4f9a. I honestly don't fully get the advantage, but now it's done!